### ISSUE TITLE
Deployment failing

### ISSUE THREAD

**Issue by johnanvik** [johnanvik/program-wars#466]
The deployment to production is failing. The relevent portion of the CI log seems to  be:

```
travis_time:end:14664267:start=1556043291970219560,finish=1556043305975096142,duration=14004876582
[0Ktravis_fold:end:before_deploy
[0Ktravis_fold:start:dpl_0
[0Ktravis_time:start:000db500
[0K$ rvm $(travis_internal_ruby) --fuzzy do ruby -S gem install dpl
Successfully installed dpl-1.10.8
Parsing documentation for dpl-1.10.8
Installing ri documentation for dpl-1.10.8
Done installing documentation for dpl after 0 seconds
1 gem installed
travis_time:end:000db500:start=1556043307425378714,finish=1556043309470632982,duration=2045254268
[0Ktravis_fold:end:dpl_0
[0Ktravis_time:start:06c24f89
[0Ktravis_fold:start:dpl.1
[33mInstalling deploy dependencies[0m
Successfully installed dpl-firebase-1.10.8
Parsing documentation for dpl-firebase-1.10.8
Installing ri documentation for dpl-firebase-1.10.8
Done installing documentation for dpl-firebase after 0 seconds
1 gem installed
/home/travis/.nvm/versions/node/v12.0.0/bin/firebase -> /home/travis/.nvm/versions/node/v12.0.0/lib/node_modules/firebase-tools/lib/bin/firebase.js

> @google-cloud/functions-emulator@1.0.0-beta.6 postinstall /home/travis/.nvm/versions/node/v12.0.0/lib/node_modules/firebase-tools/node_modules/@google-cloud/functions-emulator
> node scripts/upgrade-warning


If you're using the Emulator via the Firebase CLI, you can
disregard this message.

If you're upgrading @google-cloud/functions-emulator, these
are the recommended upgrade steps:

1.  Stop the currently running emulator, if any:

        functions stop

2.  Uninstall the current emulator, if any:

        npm uninstall -g @google-cloud/functions-emulator

3.  Install the new version of the emulator:

        npm install -g @google-cloud/functions-emulator

If you have trouble after upgrading, try deleting the config
directory found in:

    ~/.config/configstore/@google-cloud/functions-emulator

Then restart the emulator. You can also check for any renegade
Node.js emulator processes that may need to be killed:

    ps aux | grep node

+ firebase-tools@6.7.1
added 556 packages from 274 contributors in 18.803s

travis_fold:end:dpl.1
travis_fold:start:dpl.2
[33mPreparing deploy[0m

travis_fold:end:dpl.2
travis_fold:start:dpl.3
[33mDeploying application[0m

[1m[31mError:[39m[22m No project active, but project aliases are available.

Run [1mfirebase use <alias>[22m with one of these options:

  default (program-wars)
  program-wars (program-wars)
Firebase deployment failed

travis_fold:end:dpl.3
failed to deploy
```

**Comment by JaceRiehl** [comment]
I saw that build #1179 passed, is this still an issue?

**Comment by groadybones** [comment]
I am also wondering if this is still an issue or can we close this?

**Comment by johnanvik** [comment]
Fixed in ba3e5b2ae6ef8aa9cc0c923c2b5240562ad80f60
